Tests/processor.js

```diff
@@ -0,0 +1,48 @@
+const faker = require('faker');
+
+module.exports = {
+
+  generateProductId: (context, ee, next) => {
+    context.vars.productId = Math.floor(Math.random() * 1000000) + 9000000;
+    return next();
+  },
+
+  generateWait: (context, ee, next) => {
+    context.vars.wait = Math.floor(Math.random() * 20) + 3;
+    return next();
+  },
+
+  generatePageCount: (context, ee, next) => {
+    context.vars.pages = Math.floor(Math.random() * 5) + 1;
+    return next();
+  },
+
+  generateProductReview: (requestParams, context, ee, next) => {
```

Comment:
Was trying to figure out how to add json for a post, this clears it up, nicely done